numpy/random/mtrand/mtrand.pyx

```diff
@@ -3249,7 +3249,6 @@ cdef class RandomState:
         but is sharper at the peak and has fatter tails. It represents the
         difference between two independent, identically distributed exponential
         random variables.
-
```

Comment:
Oops, this blank line should not have been deleted.  I'll put it back when I update the pull request.